lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -83,6 +85,20 @@ public TaskRuntime(IInjector taskInjector, string contextId, string taskId, Hear
                 // do not rethrow since user is not required to provide name client
             }
 
+            _driverConnectionMessageHandler = new Lazy<IDriverConnectionMessageHandler>(() =>
```

Comment:
This code is in the constructor, for each instance of TaskRuntime, it will be executed one time only. And _driverConnectionMessageHandler  is an instance variable, so you do need to call GetIntance() once to assign value for it. Question is do you need Lazy<> here?